### DISCUSSION TITLE
Record a audio in .wav format with low latency using oboe

### DISCUSSION THREAD

**Discussion by prabalkanpur** [google/oboe#1404] [General]
I want to calculate the latency of my audio signal and I want to record a file in .wav format in android using oboe.
kindly help.

**Comment by philburk** [comment]
You can use OboeTester to measure latency.
https://github.com/google/oboe/tree/master/apps/OboeTester/docs

OboeTester can record a WAV file. Here is the source code. You are free to use that code in your own programs.
https://github.com/google/oboe/blob/master/apps/OboeTester/app/src/main/cpp/util/WaveFileWriter.h
